### DISCUSSION TITLE
Is it safe for gaming / AntiCheat software?

### DISCUSSION THREAD

**Discussion by tmatzxzone** [ramensoftware/windhawk#30] [Q&A]
Does the app even safe for gaming? Im scared that this app will trigger the anticheats software like riot vanguard.

**Comment by m417z** [comment]
I don't think there's a definite answer for this. Windhawk injects code into all processes in the system, including games, but there are other, legitimate tools for Windows which do that, too. I'd expect anticheat software to handle such cases gracefully, but it's impossible to know which kinds of checks and heuristics the anticheat software has without some research.

With Windhawk version 1.0, it's possible to configure a list of excluded processes, so you can exclude your games to be on the safe side. For a previous discussion about excluding processes, see https://github.com/ramensoftware/windhawk/discussions/21.
